src/test/resources/uk/gov/ons/ctp/integration/contactcentresvc/repository/impl/CaseDataRepositoryImplTest.CachedCase.json

```diff
@@ -23,6 +24,7 @@
     "postcode": "EX4 1EF",
     "addressType": "HH",
     "estabType": "Household",
-    "region": "E"
+    "region": "E",
+    "ceOrgName": "The Cached Case Organisation"
   }
```

Comment:
please add new line to end of JSON file (remove github red circle warning) for consistency

Reply:
done